_2021-finalproject/irvingskr/environment variable/check-environment.sh

```diff
@@ -0,0 +1,61 @@
+#判断环境变量是否存在
+echo -n "[Check] environment varible... "
+if [ $ROS_MASTER_URI ];then
+	echo "Pass"
+else
+	echo "Failed"
+fi
+#判断环境变量地址是否有效，将不符合格式的地址替换为空字符串，符合格式的完整保留
+echo -n "[Check] address validation...  "
```

Comment:
What will happen if the environment variable is empty

Reply:
What do you mean? Should I  check the mistake of my script or just run what you say above

Reply:
you can have a watch at this [](https://github.com/irvingskr/missing-semester/blob/0e84056fa88c04bf03362d97021c5cc8580d5272/_2021-finalproject/irvingskr/environment%20variable/run%20the%20script.png)

Reply:
_2021-finalproject/irvingskr/environment variable/run the script.png

Reply:
can I press Resolve conversation

Reply:
@MuZhou233

Reply:
certainly